src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/WindowsSettings.json

```diff
@@ -1794,6 +1800,167 @@
       "Type": "AppSettingsApp",
       "AltNames": [ "ConnectPanel", "ConnectableDevices", "ConnectWirelessAudio", "DeviceDiscovery" ],
       "Command": "ms-settings-connectabledevices:devicediscovery"
+    },
+    {
+      "Name": "AppMMC",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe" ],
+      "Command": "mmc.exe",
+      "ShowAsFirstResult" : true
+    },
+    {
+      "Name": "ActiveXControl",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe" ],
+      "Command": "mmc.exe",
+      "Note": "NoteNoMscFileExist"
+    },
+    {
+      "Name": "AuthorizationManager",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_azman" ],
+      "Command": "azman.msc"
+    },
+    {
+      "Name": "CertificatesCurrentUser",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_certmgr" ],
+      "Command": "certmgr.msc"
+    },
+    {
+      "Name": "CertificatesLocalComputer",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_certlm" ],
+      "Command": "certlm.msc"
+    },
+    {
+      "Name": "ComponentServices",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_comexp", "ComObjects" ],
+      "Command": "comexp.msc"
+    },
+    {
+      "Name": "ComputerManagement",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_compmgmt", "SystemTools", "TaskScheduler", "EventViewer", "SharedFolders", "NetworkSessions", "SMB", "LocalUsersAndGroups", "PerformanceMonitor", "DeviceManager", "PnpDevice", "Storage", "DiskManagement", "CreateAndFormatHardDiskPartitions", "GPT", "MBR", "ServicesSnapIn", "WmiControl", "WindowsManagementInstrumentation" ],
+      "Command": "compmgmt.msc"
+    },
+    {
+      "Name": "DeviceManager",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_devmgmt", "PnpDevice" ],
+      "Command": "devmgmt.msc"
+    },
+    {
+      "Name": "DiskManagement",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_diskmgmt", "Storage", "CreateAndFormatHardDiskPartitions", "GPT", "MBR" ],
+      "Command": "diskmgmt.msc"
+    },
+    {
+      "Name": "EventViewer",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_eventmgmt" ],
+      "Command": "eventmgmt.msc"
+    },
+    {
+      "Name": "LocalGroupPolicy",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_gpedit", "GroupPolicy" ],
+      "Command": "gpedit.msc"
+    },
+    {
+      "Name": "HyperVManager",
+      "Type": "AppMMC",
+      "AltNames": [ "MMC_mmcexe", "MMC_virtmgmt", "VirtualMachine" ],
+      "Command": "virtmgmt.msc"
```

Comment:
This one didn't seem to exist for me on Windows 10 or 11. Perhaps only works when the feature is turned on?
The current behavior is sort of confusing (You try to click the result and nothing happens)

Reply:
Good point.We need to add a note to the tooltip.

Reply:
Or should we remove it as it is not a built-in feature. RSAT are not added too.

Reply:
Might make sense to remove it, then?

Reply:
It'd be accessible through the Program plugin right?

Reply:
As Run Command it will be accessible using program plugin. If it has a start menu shortcut (can't verify) it will be available using `.Hyper-V Manager` too.